### ISSUE TITLE
Querying web with xpath

### ISSUE THREAD

**Issue by SebastianMpl** [nushell/nushell#9349]
### Question

I would like to ask is it possible somehow to use xpath to query web elements?
I found that query web uses css selectors and query xml uses xpath.
But when I try to query a webpage with query xml  and xpath it says "Invalid xml document".
With query web and css selectors it works fine.
Is there currently any workaround to query web with xpath?

Thank you,

### Additional context and details

#8624 - maybe related issue

**Comment by fdncred** [comment]
ya, my latest comments are in that issue. Seem like the same issue.

**Comment by amtoine** [comment]
@SebastianMpl
can you confirm this is the same as https://github.com/nushell/nushell/issues/8624? :yum:

**Comment by SebastianMpl** [comment]
@amtoine

Not quite. In #8624 the user doesn't get any data with his query. I checked the same query with different tool (xidel) and there it works and gets requested data back. 
In my case I get Invalid XML document when I try to query a webpage with query xml 'xpath' - I've read that HTML5 website are fully compliant XML document so I assumed it should work.
Don't know the degree of similarity between our issues but we're both having some issues with our queries.

**Comment by Kissaki** [comment]
Related to this: Given that `query web` currently only uses CSS selectors, it can not select DOM element content, right? (Something XPath could solve.) It can only select the full DOM element HTML. Not only its HTML or text content (`innerText` or `innerHTML`).

**Comment by WindSoilder** [comment]
Closing the issue because the question is answered.

**Comment by Kissaki** [comment]
Answered how or where?

OP asked about query web with XPath, which has not been answered.

**Comment by fdncred** [comment]
`query xml` uses xpath to query xml documents. it does not work on html pages. there is more info in the linked issue above https://github.com/nushell/nushell/issues/8624
